src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
```

Comment:
Don't use `margin` for `*` selector. You could use only `box-sizing` rule here.
If you want to reset margin for some tags you should write all of them.